### PR TITLE
Make toQuery behavior consistent with pre-0.21.7

### PR DESCRIPTION
**Pull request by kibertoad**
Accept empty schema builder.

fixes https://github.com/mikro-orm/mikro-orm/issues/961

### CODE REVIEW

**Comment by kibertoad** [comment]
@B4nan Could you take a look? I would like to release fix tonight to make sure latest knex version is fully compatible with mikro-orm.

File: lib/interface.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ module.exports = function (Target) {
   Target.prototype.toQuery = function (tz) {
     let data = this.toSQL(this._method, tz);
     if (!Array.isArray(data)) data = [data];
+    if (!data.length) {
```
Comment (kibertoad) [inline lib/interface.js:10]: Result of breakage was introduction of `reduce` after .map - `reduce` throws an error on empty array when there is no default collector.

**Comment by B4nan** [comment]
Yes I can confirm this fixed my tests too.

**Comment by kibertoad** [comment]
@lookfirst It returns an array, an empty array, if there is nothing. Changing that logic would be a veeeeery breaking change :)

**Pull request merged**